src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+const ROOT_URL='https://yts.am/api/v2/list_movies.json';
+
+export const SEARCH_MOVIES='SEARCH_MOVIES';
+export const POPULAR_DOWNLOADS='POPULAR_DOWNLOADS';
+export const LATEST_TORRENTS='LATEST_TORRENTS';
+
+export function SearchMovies(movieTerm) {
+  const request= axios.get(`${ROOT_URL}?limit=5&query_term=${movieTerm}`);
+  return {
+    type: SEARCH_MOVIES,
+    payload: request
+  };
+};
+
+export function getPopularDownloads() {
+  const request = axios.get(`${ROOT_URL}?limit=4&sort_by=download_count`);
```

Comment:
Also here add limit and sort_by to function parameters:
getPopularDownloads({limit = 8, sort_by = 'download_count'} = {})